_data/simple-icons.json

```
@@ -2384,6 +2384,12 @@
             "hex": "5881D8",
             "source": "https://commons.wikimedia.org/wiki/File:Clojure_logo.svg"
         },
+        {
+            "title": "ClojureScript",
+            "hex": "96CA4B",
+            "source": "https://github.com/oakmac/cljs-logo/blob/acc181dc6885dbf9d7b6046e6d21b46d8f4238eb/cljs-outline.svg",
```

```suggestion
            "source": "https://github.com/oakmac/cljs-logo/blob/acc181dc6885dbf9d7b6046e6d21b46d8f4238eb/cljs.svg",
```

Should we instead use the icon that features the text `cljs`? To me, that's more identifyable.

---

I did it because of this line in Readme
https://github.com/oakmac/cljs-logo/blob/acc181dc6885dbf9d7b6046e6d21b46d8f4238eb/README.md?plain=1#L55-L57
I will still update it